entries/b/boost.com.au.json

```
@@ -0,0 +1,19 @@
+{
+  "Boost Mobile": {
```

```suggestion
  "Boost Mobile [AU]": {
```

The U.S. version of Boost Mobile is entirely separate to my knowledge